### ISSUE TITLE
About the list.txt and generators.py

### ISSUE THREAD

**Issue by codefwer** [voxelmorph/voxelmorph#430]
![image](https://user-images.githubusercontent.com/79414489/168292856-eed8667b-684b-451c-808d-3d5c9fe7491b.png)
i am sorry that i really dont konw what the `list.txt` really is.  Is the moving image's path and fixed image's path in it? Can you give me a demo for the `list.txt` ,please

![image](https://user-images.githubusercontent.com/79414489/168293969-39b9a024-3920-4c15-b436-50e62ef87a5c.png)
i also dont understand what's the` scan1` and `scan2`,  and as i konw the `dataloader` is important  in training by using pytorch, why don't have it in `scripts/torch/train.py`
i am soory to ask so many questions, but they really confuse me a lot

**Comment by adalca** [comment]
list.txt is a file of full paths to all your training images, one per line. @ahoopes could confirm.

**Comment by codefwer** [comment]
> 

the moving images' path and fixed images' path are all in one list.txt?

**Comment by adalca** [comment]
Yes. you just have a list of N images, and at each iteration the loader randomly samples one to be 'moving' and another to be 'fixed'

**Comment by codefwer** [comment]
why randomly samples one to be 'moving'  ,i really dont konw that.  Could it be that we use the konwn 'moving' image and 'fixed' image and the first one is really 'moving' image ,the second is the corresponding fixed image?

**Comment by adalca** [comment]
it depends on what you are doing, but if you are trying to register images of different subjects, then randomly sampling images to be 'moving' and 'fixed' teaches the network to be able to register any image to any other image

**Comment by codefwer** [comment]
> it depends on what you are doing, but if you are trying to register images of different subjects, then randomly sampling images to be 'moving' and 'fixed' teaches the network to be able to register any image to any other image

thanks a lot ,i think i need some time to do some experiments, could you please don't close this issue now,?
